vendor/bat-native-ads/src/bat/ads/internal/features/text_embedding_features.cc

```diff
@@ -0,0 +1,49 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/features/text_embedding_features.h"
+
+#include "base/metrics/field_trial_params.h"
+
+namespace ads {
+namespace targeting {
+namespace features {
+
+namespace {
+
+constexpr char kFeatureName[] = "TextEmbedding";
+
+constexpr char kFieldTrialParameterPageEmbeddingsHistorySize[] =
+    "text_embeddings_history_size";
+const int kDefaultTextEmbeddingsHistorySize = 10;
+
+constexpr char kFieldTrialParameterResourceVersion[] =
+    "text_embedding_resource_version";
+constexpr int kDefaultResourceVersion = 1;
+
+}  // namespace
+
+const base::Feature kTextEmbedding{kFeatureName,
```

Comment:
this should not be enabled by default, right?

Reply:
switched to `base::FEATURE_DISABLED_BY_DEFAULT`